scripts/scenario_29_surf.lua

```diff
@@ -0,0 +1,1762 @@
+-- Name: Surf's Up!
+-- Description: Wave after wave of enemy ships attack, each wave harder than the previous 
+--- wave. Single or multiple player ships may participate. Scenario is over when the
+--- friendly bases are destroyed. Loosely based on the Waves scenario.
```

Comment:
I think it would be good to mention that this is a more complicated scenario then the "Waves" scenario, where waves is quite basic, this provides different challenges.

We should update descriptions so they are helpful for people to know to pick a certain scenario over another for a particular crew.

Reply:
I updated the description. I think it's available as part of this pull request

Reply:
About that: would it be possible to integrate the old waves scenario into this? Like, in a similar fashion Basic battle and Quick Basic merged a while ago? Haven't played it yet, but from the looks of it, the default settings of this ones seem to be similar to the default setting of waves. The playable area also seems to have a similar scope. Or is there something that makes both scenarios incompatible to each other?

Reply:
It's quite possible. I think I tried making version of Waves with more options a couple of years ago that did not get accepted. The changes in Surf's Up are different than what I did before.